KMaina/Fast-Food-App
# #160748647 Allow an Admin to Add a Food Item

#### What does this PR do?

Allows an admin to add a food item

#### Description of Task to be completed?

The admin should add a food item

#### How should this be manually tested?

- Clone the repo onto your local machine.
- Open a terminal and run cd Fast-Food-App
- run `git clone https://github.com/KMaina/Fast-Food-App.git`
- run `git checkout ch-admin-add-food-160748647`
- In the folder where you cloned the repo, you should open the UI folder.
- Open the admin-add-food.html to see it in your browser for review.

#### Any background context you want to provide?

This is a redesign of the page

#### What are the relevant pivotal tracker stories?

#160748647
